cfgov/unprocessed/js/organisms/FormSubmit.js

```diff
@@ -161,12 +158,9 @@ function FormSubmit( element, baseClass, opts ) {
     }
 
     function fadeInMessage() {
-      if ( opts.minReplacementHeight ) {
-        _baseElement.style.marginBottom = Math.min( _formElement.offsetHeight, 100 ) + 'px';
-      }
-      _formElement.style.display = 'none';
       _notification.setTypeAndContent( _notification.SUCCESS, message );
       _notification.show();
+      _baseElement.replaceChild( _notification.getElement(), _formElement );
```

Comment:
What do think about using `_notificationElement` in place of `_notification.getElement()` and leaving the notification API alone?

Reply:
Hmm, yes -- it looks like `_notificationElement` is already defined in this module, so that would make sense.